content/6_3_simulation-general.html

```diff
@@ -409,7 +409,7 @@ <h5>Review Tips</h5>
                                 </div>
                                 <div class="row">
                                     <div class="col-12 border-bottom border-left border-right">
-                                        <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#imageModal" data-full="../assets/images/ashrae-prm_ecb-compliance-report.png">
```

Comment:
it looks like you changed the image that was being used for this button but the image you changed it to is also applicable, so no big deal.

When you go back to complete this table with all of the relevant buttons you will need to do add the old one back again as a new button.

Reply:
ashrae-prm_ecb-compliance-report.png